nrf_security/Kconfig.psa

```diff
@@ -80,7 +80,14 @@ endif # PSA_CRYPTO_DRIVER_OBERON
 menuconfig PSA_CRYPTO_DRIVER_BUILTIN
 	bool
 	prompt "PSA Builtin" if !PSA_PROMPTLESS
-	default y if !CRYPTOCELL_USABLE
+	# Fix for NCSDK-13536: Enable builtin if nrf_oberon only due to lacking support
+	default y if PSA_CRYPTO_DRIVER_OBERON && !PSA_CRYPTO_DRIVER_CC3XX && \
+		     (PSA_WANT_ALG_CBC_PKCS7 || PSA_WANT_ALG_CMAC || \
+		      PSA_WANT_ALG_HMAC || PSA_WANT_ALG_HKDF)
+	# nrf_oberon lacks RSA support
```

Comment:
does this mean that PSA Builtin must be selected when `nrf_oberon` is selected, or how is this supposed to be understood ?

This only changes the default, meaning a user or project selecting Oberon can still disable this, is that allowed ?

And how about the stuck symbol syndrome when user changes the backend selection for testing.

Reply:
Builtin is currently enabled just because the drivers are not fully featured, something we plan on fixing as soon as we can. There is a possibility of stuck symbols, but we deem this fairly uneventful, as it only potentially leads to an increase in code-size. 

If you turn of the masked items (RSA configurations using PSA_WANT_ALG_RSA_XXXX then it doesn't matter if you enable the builtin)